.travis_scripts/create_feedstocks.py

```diff
@@ -83,6 +83,7 @@ def print_rate_limiting_info(gh):
     print("")
     print("GitHub API Rate Limit Info:")
     print("---------------------------")
+    print("user: ", gh.get_user().login)
```

Comment:
This would fail if there's no requests left.

Reply:
I was trying to be clever. Hold on.

Reply:
Alright this should be fixed.